src/renderer/common/components/toast/Toast.tsx

```diff
@@ -185,7 +191,46 @@ export class Toast extends React.Component<IProps, IState> {
                             // ignore
                         }
                     }
-                }>{ this.props.message }</p>
+                }>{this.props.publicationTitle ?
+                    <span>
+                        {this.props.message}
+                        <br/>
+                        <Link
+                            style={{textDecoration: "underline", fontStyle:"italic", fontWeight: "bold", color: "var(--color-primary)"}}
+                            to={{
+                                ...location,
+                                pathname: "/library",
+                                search: `?focus=search&value=${this.props.publicationTitle}`,
```

Comment:
This can match multiple publications, I wonder if we could be more specific? (uuid?)